### DISCUSSION TITLE
Double door open

### DISCUSSION THREAD

**Discussion by alfredcort** [Zrips/CMI#6901] [Enhancement]
I would love to see an feature to see double doors opening when clicking only 1 door.

Eg when clicking the left door, the right door will also open.

**Comment by CloudeLecaw** [comment]
This is something protection Plugins can do by flags like Redprotect does.

So i see this more as a new flag suggeestion for Zrips plugin Ressidence. As its more an aspect of optional funtion if doors can be opened or not.
